packages/dd-trace/src/appsec/remote_config/manager.js

```diff
@@ -33,6 +34,10 @@ class RemoteConfigManager extends EventEmitter {
       port: config.port
     }))
 
+    const tags = { ...config.tags }
+    tags[GIT_REPOSITORY_URL] = config.repositoryUrl
+    tags[GIT_COMMIT_SHA] = config.commitSHA
```

Comment:
less HiddenClass created ?
```suggestion
    const tags = {
      ...config.tags,
      [GIT_REPOSITORY_URL]: config.repositoryUrl,
      [GIT_COMMIT_SHA]: config.commitSHA
    }
```

Reply:
Fixed 👍